the-collab-lab/tcl-30-whats-near-me
# [TCL30-51] Colors & Typography

## Description

As a user

I would like the colors and typography to be unified across the application

so that using the app feels like a unified experience, I’m not confused, and I understand the brand.

## Related Issue

[TCL30-51](https://the-collab-lab.atlassian.net/browse/TCL30-51)

## Acceptance Criteria

- Implement Exo as the typeface for Headings
- Implement Merriweather Sans for body text
- Implement #264653 as primary color
- Implement #2A9D8F as secondary
- #fff as accent
- #000 for and #626262 for fonts

Use CSS variables.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Update

- Add typography

```css
/* Typography */
@import url('https://fonts.googleapis.com/css2?family=Exo:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Merriweather+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
```

- Create CSS variables

```css
/* CSS Variables */
:root {
  --font-family-headings: 'Exo', sans-serif;
  --font-family-body-text: 'Merriweather Sans', sans-serif;
  --primary-color: #264653;
  --secondary-color: #2a9d8f;
  --accent-color: #fff;
  --primary-text-color: #000;
  --secondary-text-color: #626262;
  --shadows-color: --secondary-text-color;
}
```

## Testing Steps / QA Criteria

- Open the `App.css` file
- Go to the top of the file and you should see two sections: Typography & CSS Variables.
